dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/OASValidator.java

```diff
@@ -80,6 +89,21 @@ public boolean addOperationId(String operationId) {
         return !operationIds.add(operationId);
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public void addLinkOperationId(String operationId, String location) {
+        linkOperationIds.put(operationId, location);
+    }
+
+    public void validateLinkOperationIds() {
+        for (String k : linkOperationIds.keySet()) {
+            if (!operationIds.contains(k)) {
+                final String message = Tr.formatMessage(tc, "linkOperationIdInvalid", k);
+                addValidationEvent(new ValidationEvent(Severity.ERROR, linkOperationIds.get(k), message));
```

Comment:
After the above mentioned modifications to the Map, ensure that the value retrieved is correctly converted into String.